db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SubqueryNode.java

```diff
@@ -1386,6 +1386,209 @@ public void generateExpression(ExpressionClassBuilder expressionBuilder,
             mbex.pushThis();
             mbex.callMethod(VMOpcode.INVOKEVIRTUAL,null,mb.getName(),subqueryTypeString,0);
         }
+
+        generateSubqueryResultSet(expressionBuilder);
+    }
+
+    public void generateSubqueryResultSet(ExpressionClassBuilder expressionBuilder) throws StandardException {
+        CompilerContext cc=getCompilerContext();
```

Comment:
This method is very similar to generateExpression(), would it be possible to extract the commonalities so that there isn't so much copy paste?

Reply:
👍

Reply:
Refactored and extracted common code.